src/pages/pass-finalisation/pass-finalisation.ts

```diff
@@ -68,9 +84,8 @@ export class PassFinalisationPage extends PracticeableBasePageComponent {
   passCertificateCtrl: string = 'passCertificateNumberCtrl';
   @ViewChild('passCertificateNumberInput')
   passCertificateNumberInput: ElementRef;
-
   inputSubscriptions: Subscription[] = [];
-
+  testOutcome: string = '1';
```

Comment:
This feels a bit hard coded , why do we need this and why do we need to pass it into the debrief witnessed field.

If it is needed can we change it to ActivityCodes.Pass instead of using a string.

Reply:
It is just used to drive the visibility of the field because this will be reused on each of the pass, fail journeys and each termination code. I've changed it to `ActivityCodes.Pass`